phpseclib/Crypt/Hash.php

```diff
@@ -173,27 +170,27 @@ function setHash($hash)
     /**
      * Compute the HMAC.
      *
-     * @access public
      * @param string $text
      * @return string
      */
-    function hash($text)
+    public function hash($text)
     {
         $output = !empty($this->key) || is_string($this->key) ?
             hash_hmac($this->hash, $text, $this->key, true) :
             hash($this->hash, $text, true);
 
-        return strlen($output) > $this->l ? substr($output, 0, $this->l) : $output;
+        return strlen($output) > $this->length
+            ? substr($output, 0, $this->length)
+            : $output;
     }
 
     /**
      * Returns the hash length (in bytes)
      *
-     * @access public
```

Comment:
access was public anyway - i think we should remove these annotations on 2.x too